src/application/content-actions/index.js

```diff
@@ -31,29 +39,56 @@ const ContentActions = {
             actions: applicationStore.getActions() || { primary: [], secondary: [] }
         };
     },
+
     /**
      * Component change handler
      */
     _handleComponentChange() {
         this.setState(this._getStateFromStore());
     },
-    /** @inheriteddoc */
+
+    /** @inheritdoc */
     render() {
-        const { actions } = this.state;
+        const { actions: { primary, secondary } } = this.state;
         return (
             <div className={this._getStyleClassName()} data-focus='content-actions'>
-                {actions.primary.map((primary) => {
+                {primary.map((primary) => {
                     if (Array.isArray(primary.action)) {
                         return (
-                            <Dropdown iconProps={{ name: primary.icon }} operationList={primary.action} shape='fab' />
+                            <Dropdown
+                                iconProps={{ name: primary.icon, iconLibrary: primary.iconLibrary }}
+                                operationList={primary.action}
+                                shape='fab'
+                            />
                         );
                     } else {
                         return (
-                            <Button handleOnClick={primary.action} icon={primary.icon} label={primary.label} shape='fab' style={{ className: primary.className }} type='button' />
+                            <Button
+                                handleOnClick={primary.action}
+                                icon={primary.icon}
+                                iconLibrary={primary.iconLibrary}
+                                label={primary.label}
+                                shape='fab'
+                                style={{ className: primary.className }}
+                                type='button'
+                            />
                         );
                     }
                 })}
-                <Dropdown iconProps={{ name: 'more_vert' }} operationList={actions.secondary} shape='fab' />
+                {isArray(secondary) && (
```

Comment:
Basically, the code for rendering secondary or primary actions should be the same.
Can you make a renderActions function, and use it for both ?